java/aquality-selenium-template/src/test/resources/settings.json

```diff
@@ -0,0 +1,79 @@
+{
+  "browserName": "chrome",
+  "isRemote": false,
+  "remoteConnectionUrl": "http://localhost:4444/wd/hub",
+  "isElementHighlightEnabled": true,
+
+  "driverSettings": {
+    "chrome": {
+      "webDriverVersion": "latest",
+      "capabilities": {
+        "enableVNC": true
+      },
+      "options": {
+        "intl.accept_languages": "en",
+        "safebrowsing.enabled": "true",
+        "profile.default_content_settings.popups": "0",
+        "disable-popup-blocking": "true",
+        "download.prompt_for_download": "false",
+        "download.default_directory": "//home//selenium//downloads"
+      },
+      "startArguments": [
+        "window-size=1920,1080",
+        "--no-sandbox",
+        "--disable-popup-blocking",
+        "--disable-gpu"
+      ]
+    },
+    "firefox": {
+      "webDriverVersion": "latest",
+      "capabilities": {
+        "enableVNC": true
+      },
+      "options": {
+        "intl.accept_languages": "en",
+        "browser.download.dir": "//home//selenium//downloads",
```

Comment:
can we specify a relative download directory? Do we plan to create a pipeline to check that test works?

Reply:
I'm not sure that it is necessary to create tests for each features from the lib. So, my answer is no.